.changeset/techdocs-buttons-film.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-techdocs': patch
+---
+
+Fix `EntityDocs` component to use objects instead of `<Route>` elements, otherwise "outlet" will be null on sub-pages and add-ons won't render.
```

Comment:
`EntityTechdocsContent` right? Perhaps this description is a bit too low level as well? Thinking we could just describe that it fixed an issue with adding addons in nested routes

Reply:
Fixed!